env/production/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 bases:
-  - github.com/cds-snc/notification-manifests//base?ref=v0.26.0
+  - github.com/cds-snc/notification-manifests//base?ref=v0.28.0
```

Comment:
Is there a specific reason why we're note using 0.27 here?

Reply:
Splitted it in two PRs, we can release 0.27 first https://github.com/cds-snc/notification-manifests/pull/256